scripts/v8-to-v9.py

```
@@ -0,0 +1,178 @@
+    parser.add_argument("--output", "-o", required=True, help="Path to output upgraded file")
+    parser.add_argument("--blank", "-n", required=False, help="Path to blank genesis file for upgrade reference")
+    parser.add_argument("--dogfood_addr", "-d", required=True, help="Dogfood address for testnet")
+    parser.add_argument("--chain_id", "-c", required=True, help="Chain id")
+    args = parser.parse_args()
+
+    # Read input JSON
+    with open(args.input, 'r') as f:
+        genesis_data = json.load(f)
+
+    # Upgrade the file
+    genesis_data = upgrade_genesis(genesis_data, args.blank, args.dogfood_addr,args.chain_id)
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)